Unable to initiate run for quarkus-reactive-mysql-pool-client

Since 21/03 we are getting:

```
Attempting to launch CI for quarkiverse/quarkus-reactive-mysql-pool-client/
Commited context changes for quarkiverse/quarkus-reactive-mysql-pool-client/
Unexpected http response status (404) received when starring repository quarkiverse/quarkus-reactive-mysql-pool-client/
Error: Process completed with exit code 1.
```

See [this](https://github.com/quarkusio/quarkus-ecosystem-ci/actions/runs/4475689262) run for example.

@gastaldi any idea what's going on?

--- a/quarkiverse-reactive-mysql-pool-client/info.yaml
+++ b/quarkiverse-reactive-mysql-pool-client/info.yaml
@@ -1,4 +1,4 @@
-url: https://github.com/quarkiverse/quarkus-reactive-mysql-pool-client/
+url: https://github.com/quarkiverse/quarkus-reactive-mysql-pool-client
 issues:
   repo: quarkiverse/quarkiverse
   latestCommit: 104

